virtualenv.py

```diff
@@ -354,11 +354,21 @@ def copyfile(src, dest, symlink=True):
         logger.info('Copying to %s', dest)
         copyfileordir(src, dest, symlink)
 
-def writefile(dest, content, overwrite=True):
+def get_system_encoding():
+    if is_win:
+        # .encode('mbcs') is broken on Windows. For example,
+        # u'\u00a3'.encode('mbcs') gives a wrong result in CP437 (en-US), and
```

Comment:
What do you mean a wrong result? u"\u00a3" (£ sign) is encoded to \x9c in CP437. Is that not what you get? Note that "mbcs" is the ANSI codepage, which is *not* the same code page as GetOEMCP.